### ISSUE TITLE
India Scenario Date Change

### ISSUE THREAD

**Issue by SM-ins** [lisphilar/covid19-sir#129]
Dear Lisphilar,

i need one help..,please go in your kaggle notebook of latest version..,go into the scenario section of India,see the output of ind_scenario.trend(),on the top of the graph you will find the starting date is from  22 May..,can it be shifted to March 20 or 22 March..Please do it if possible or let me know I will try to do it by myself.

With regards,
Rakesh

**Comment by lisphilar** [comment]
Dear Rakesh,
This may be related to #126 
Records to 21May2020 are registered as initial (0th) phase and removed to avoid low accuracy of estimation because the number of cases are small.

We can include the records from 20Mar2020 to 21May2020 with the following codes.

```Python
scenario = cs.Scenario(jhu_data, population_data, country="India")
# Set phases with S-R trend analysis and keep 0th phase
scenario.trend(set_phases=True, include_init_phase=True)
# Separate 0th phase on 20Mar2020
scenario.separate(date="20Mar2020", phase="0th")
# Remove 0th phase (to 19Mar2020)
scenario.delete(phases=["0th"])
# Summarize
scenario.summary()
scenario.trend(set_phases=False)
```

**Comment by lisphilar** [comment]
This will be solved in #126
